docker-compose.image.example.yaml

```diff
@@ -1,75 +1,208 @@
+      - GITHUB_CLIENT_SECRET=$GITHUB_CLIENT_SECRET
+      - FRONTEND_URL=$FRONTEND_URL
+      - ENCRYPTION_KEY=$ENCRYPTION_KEY
+      - WEAVIATE_USE_EMBEDDED=$WEAVIATE_USE_EMBEDDED
+      - GOOGLE_API_KEY=$GOOGLE_API_KEY
+      - SEARCH_ENGINE_ID=$SEARCH_ENGINE_ID
+      - SERP_API_KEY=$SERP_API_KEY
+      - EMAIL_ADDRESS=$EMAIL_ADDRESS
+      - EMAIL_PASSWORD=$EMAIL_PASSWORD
+      - EMAIL_SMTP_HOST=$EMAIL_SMTP_HOST
+      - EMAIL_SMTP_PORT=$EMAIL_SMTP_PORT
+      - EMAIL_IMAP_SERVER=$EMAIL_IMAP_SERVER
+      - EMAIL_SIGNATURE=$EMAIL_SIGNATURE
+      - EMAIL_DRAFT_MODE_WITH_FOLDER=$EMAIL_DRAFT_MODE_WITH_FOLDER
+      - EMAIL_ATTACHMENT_BASE_PATH=$EMAIL_ATTACHMENT_BASE_PATH
+      - GITHUB_USERNAME=$GITHUB_USERNAME
+      - GITHUB_ACCESS_TOKEN=$GITHUB_ACCESS_TOKEN
+      - JIRA_INSTANCE_URL=$JIRA_INSTANCE_URL
+      - JIRA_USERNAME=$JIRA_USERNAME
+      - JIRA_API_TOKEN=$JIRA_API_TOKEN
+      - SLACK_BOT_TOKEN=$SLACK_BOT_TOKEN
+      - STABILITY_API_KEY=$STABILITY_API_KEY
+      - ENGINE_ID=$ENGINE_ID
+      - RESOURCE_VECTOR_STORE=$RESOURCE_VECTOR_STORE
+      - RESOURCE_VECTOR_STORE_INDEX_NAME=$RESOURCE_VECTOR_STORE_INDEX_NAME
+      - REDIS_VECTOR_STORE_URL=$REDIS_VECTOR_STORE_URL
+      - QDRANT_PORT=$QDRANT_PORT
+      - QDRANT_HOST_NAME=$QDRANT_HOST_NAME
+      - CHROMA_HOST_NAME=$CHROMA_HOST_NAME
+      - CHROMA_PORT=$CHROMA_PORT
     command: ["/app/entrypoint_celery.sh"]
     volumes:
-      - "./workspace:/app/workspace"
+      - ./workspace:/app/workspace
 
   gui:
     image: "superagidev/superagi-frontend:main"
+    restart: unless-stopped
     environment:
       - NEXT_PUBLIC_API_BASE_URL=/api
-    networks:
-      - super_network
 
-  super__redis:
+  redis:
     image: "redis/redis-stack-server:latest"
-    networks:
-      - super_network
-#    uncomment to expose redis port to host
-#    ports:
-#      - "6379:6379"
+    restart: unless-stopped
+    healthcheck:
+      test: ["CMD", "redis-cli", "ping"]
+      interval: 1s
+      timeout: 3s
+      retries: 30
+    #    uncomment to expose redis port to host
+    #    ports:
+    #      - "6379:6379"
     volumes:
-      - redis_data:/data
+      - superagi_redis_data:/data
 
-  super__postgres:
+  postgres:
     image: "docker.io/library/postgres:latest"
+    restart: unless-stopped
     environment:
-      - POSTGRES_USER=superagi
-      - POSTGRES_PASSWORD=password
-      - POSTGRES_DB=super_agi_main
+      - POSTGRES_USER=$DB_USERNAME
+      - POSTGRES_PASSWORD=$DB_PASSWORD
+      - POSTGRES_DB=$DB_NAME
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready", "-d", "$DB_NAME"]
+      interval: 5s
+      timeout: 60s
+      retries: 10
+      start_period: 10s
     volumes:
       - superagi_postgres_data:/var/lib/postgresql/data/
-    networks:
-      - super_network
-#    uncomment to expose postgres port to host
-#    ports:
-#      - "5432:5432"
+  #    uncomment to expose postgres port to host
+  #    ports:
+  #      - "5432:5432"
+
+  weaviate:
```

Comment:
can we remove weaviate setup from the main example,
you can add an another example with weaviate setup, so others can also contribute to any vector-store they are using
you can add a docker-compose.image.example-weaviate.yaml, so community members wantting to use weaviate can refer it

Reply:
Sure thing, my reasoning was that Weaviate is the only supported self-hosted vector store, but now I'm realizing that Qdrant is an alternative so this makes perfect sense.